src/DGtal/io/readers/ITKReader.ih

```diff
@@ -64,7 +64,8 @@ namespace DGtal {
     const Domain& domain = dgtal_itk_image.domain();
 
     typedef ConstImageAdapter<DGtalITKImage, Domain, functors::Identity, Value, Functor> AdaptedImage;
-    const AdaptedImage adapted(dgtal_itk_image, domain, functors::Identity(), aFunctor);
+    const functors::Identity identityFunctor{};
```

Comment:
:smile: 
In that case, it is exactly the same as without `{}`: it calls the constructor without argument. It is a new syntax from `C++11`. It enhances the construction of object and, especially, it allows me to replace a common mistake that I make:
```const functors::Identity identityFunctor();```
by a valid syntax:
```const functors::Identity identityFunctor{};```
:octocat: